src/MAUI/Maui.Samples/Samples/Data/EditFeaturesUsingFeatureForms/EditFeaturesUsingFeatureForms.xaml.cs

```diff
@@ -0,0 +1,160 @@
+            name: "Edit features using feature forms",
+            category: "Data",
+            description: "Display and edit feature attributes using feature forms.",
+            instructions: "Tap a feature on the map to open a sheet displaying the feature form. Select form elements in the list and perform edits to update the field values. Tap the submit icon to commit the changes on the web map.",
+            tags: new[] { "edits", "feature", "featureforms", "form", "toolkit" })]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData("516e4d6aeb4c495c87c41e11274c767f")]
+    public partial class EditFeaturesUsingFeatureForms : ContentPage
+    {
+        public EditFeaturesUsingFeatureForms()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            try
+            {
+                // Create the ArcGIS Online portal.
+                ArcGISPortal portal = await ArcGISPortal.CreateAsync();
+                // Get the Naperville water web map item using its ID.
+                PortalItem webmapItem = await PortalItem.CreateAsync(portal, "516e4d6aeb4c495c87c41e11274c767f");
+                // Create a map from the web map item.
+                Map onlineMap = new Map(webmapItem);
+                // Display the map in the MapView.
+                MyMapView.Map = onlineMap;
+            }
+            catch (Exception ex)
+            {
+                await DisplayAlert("Error", ex.Message, "OK");
+            }
+        }
+
+        private async void MyMapView_GeoViewTapped(object sender, Esri.ArcGISRuntime.Maui.GeoViewInputEventArgs e)
+        {
+            try
+            {
+                // Perform identify operation to get the feature
+                IdentifyLayerResult identifyResult = await MyMapView.IdentifyLayerAsync(MyMapView.Map.OperationalLayers.First(), e.Position, 12, false);
+                ArcGISFeature feature = identifyResult.GeoElements.OfType<ArcGISFeature>().FirstOrDefault();
+                if (feature != null)
+                {
+                    // Create a feature form
+                    var featureForm = new FeatureForm(feature);
+
+                    // Create a feature form view
+                    var featureFormView = new FeatureFormView
+                    {
+                        FeatureForm = featureForm,
+                        Padding = new Thickness(10),
+                        VerticalScrollBarVisibility = ScrollBarVisibility.Default,
+                    };
+
+                    var saveButton = new Button
+                    {
+                        Text = "Save",
+                    };
+
+                    saveButton.Clicked += async (s, e) =>
+                    {
+                        // Check if there are validation errors
+                        if (featureForm.ValidationErrors.Any())
+                        {
+                            return;
```

Comment:
perhaps show a dialog with the errors?
You could also bind the save button's IsEnabled status to the forms' IsValid property.